src/KingSystem/Event/evtInfoData.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <KingSystem/Resource/resLoadRequest.h>
+#include <KingSystem/Utils/Byaml/Byaml.h>
+#include <heap/seadDisposer.h>
+#include <prim/seadSafeString.h>
+
+namespace ksys::evt {
+
+class InfoDataUnknownType {
+public:
+    virtual ~InfoDataUnknownType();
+};
+
+class InfoData {
+    SEAD_SINGLETON_DISPOSER(InfoData)
+    InfoData();
+    virtual ~InfoData();
+
+public:
+    bool getEntry(al::ByamlIter* iter, const sead::SafeString& eventFlowName,
+                  const sead::SafeString& entryPoint);
+    bool getSceneChangeEventTraverseLimit(const sead::SafeString& eventFlowName);
+    void init(sead::Heap* heap);
+
+private:
+    void doInit(sead::Heap* heap, OverlayArena* a3);
+    al::ByamlIter* rootIter = nullptr;
+    ksys::res::Handle res2;
+    InfoDataUnknownType* field_80 = nullptr;
```

Comment:
The convention is to name unknown fields by their hexadecimal offset (with letters in lowercase), without any "field" prefix:
```suggestion
    InfoDataUnknownType* _80 = nullptr;
```